tests/util/pool_tests.rs

```diff
@@ -0,0 +1,66 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+/// Test the functionality of stratis pools.
+extern crate devicemapper;
+extern crate env_logger;
+
+use std::fs::OpenOptions;
+use std::io::Write;
+use std::path::Path;
+
+use self::devicemapper::DM;
+use self::devicemapper::Sectors;
+use self::devicemapper::consts::SECTOR_SIZE;
+
+use libstratis::engine::Pool;
+use libstratis::engine::strat_engine::device::wipe_sectors;
+use libstratis::engine::strat_engine::metadata::BDA_STATIC_HDR_SECTORS;
+use libstratis::engine::strat_engine::pool::{DATA_BLOCK_SIZE, DATA_LOWATER, INITIAL_DATA_SIZE,
+                                             StratPool};
+use libstratis::engine::strat_engine::StratEngine;
+use libstratis::engine::types::Redundancy;
+
+/// Verify that a the physical space allocated to a pool is expanded when
+/// the nuber of sectors written to a thin-dev in the pool exceeds the
+/// INITIAL_DATA_SIZE.  If we are able to write more sectors to the filesystem
+/// than are initially allocated to the pool, the pool must have been expanded.
+pub fn test_thinpool_expand(paths: &[&Path]) -> () {
+    for path in paths {
```

Comment:
This should be taken care of in the test setup infrastructure, so it should be unneeded here.